app/src/main/java/org/wikipedia/suggestededits/Contribution.kt

```diff
@@ -0,0 +1,25 @@
+package org.wikipedia.suggestededits
+
+import org.wikipedia.dataclient.WikiSite
+import java.util.*
+
+class Contribution internal constructor(val qNumber: String, var title: String, var description: String, val editType: Int, var imageUrl: String,
+                                        val date: Date, val wikiSite: WikiSite, var pageViews: Long, var tagCount: Int) {
```

Comment:
It looks like the `tagCount` field is written, but not used anywhere.  What's the plan for this field?

Reply:
to be used in the details screen [used in #1185 ]